src/crimson/os/seastore/segment_manager/zns.cc

```diff
@@ -62,6 +62,10 @@ static zns_sm_metadata_t make_metadata(
   size_t segment_size = zone_size;
   size_t zones_per_segment = segment_size / zone_size;
   size_t segments = (num_zones - RESERVED_ZONES) / zones_per_segment;
+  size_t available_size = zone_capacity * segments;
+
+  assert(size == num_zones * zone_size);
+  size = available_size;
```

Comment:
`size` is a vague name for zns. rename `size` to `total_size`, and use `available_size` in this method below (or any other better names).

Reply:
Just to confirm, do you want to change `size` to `total_size` only in this function `make_metadata()`
or 
change `size` to `total_size` in `struct zns_sm_metadata_t` (then change it in all places)

Reply:
We can fix `struct zns_sm_metadata_t` until really needed. Fixing the name in the function should be fine for now, I think.

Reply:
Ok.